docs/opentelemetry-tracing.rst

```diff
@@ -9,6 +9,7 @@ To take advantage of these traces, we first need to install OpenTelemetry:
 .. code-block:: sh
 
     pip install opentelemetry-api opentelemetry-sdk opentelemetry-instrumentation
+    pip install opentelemetry-exporter-google-cloud
 
     # [Optional] Installs the cloud monitoring exporter, however you can use any exporter of your choice
     pip install opentelemetry-exporter-google-cloud
```

Comment:
This package is deprecated and will not receive any more updates. Also, this line is already available below and is a duplicate here.
https://pypi.org/project/opentelemetry-exporter-google-cloud/

Replace this with `pip install opentelemetry-exporter-gcp-trace`